coverage/.resultset.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
you have to remove the coverage files from the git cache.
http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

Reply:
done